test/libsolidity/semanticTests/functionTypes/function_external_delete_storage.sol

```diff
@@ -0,0 +1,39 @@
+contract C {
+    function() external public x;
+    uint public y = 0;
+
+    function increment() public {
+        ++y;
+    }
+
+    function set() external {
+        x = this.increment;
+    }
+
+    function incrementIndirectly() public {
+        x();
+    }
+
+    function deleteFunction() public {
+        // used to lead to an ICE during IR
+        delete x;
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// x() -> 0
+// y() -> 0
+// increment() ->
+// y() -> 1
+// set() ->
+// x() -> 7175878113405833249322534082293024008058894263826350527301804152930743353344
```

Comment:
What does this mean when a bytes32 is returned when a void function is invoked in our test framework?

Reply:
It's the hex "0xfdd67305928fcac8d213d1e47bfa6165cd0b87bd09de08a0000000000000000". The address and selector. Maybe I'll just change it to hex for better readability.